harness/determined/deploy/gke/cli.py

```diff
@@ -224,6 +225,7 @@ def configure_helm(args: argparse.Namespace) -> None:
     helm_dir = Path(args.helm_dir)
     with (helm_dir / "Chart.yaml").open() as f:
         helm_chart = safe_load_yaml_with_exceptions(f)
+    warn_version_mistmatch(args.det_version)
```

Comment:
I think this is the only one that doesn't check for `args.det_version` being `None`?
e.g., 
```
    if args.det_version is None:
        args.det_version = determined.__version__
```

Reply:
aws/cli doesn't either.

I added that only to the ones that had a default in the cli arg definition to keep the behavior the same as before.